python/tk_desktop/desktop_window.py

```diff
@@ -843,9 +847,7 @@ def slide_finished():
         anim_group.start()
 
     def open_site_in_browser(self):
-        login = ShotgunLogin.get_instance_for_namespace("tk-desktop")
-        connection = login.get_connection()
-        url = connection.base_url
+        url = shotgun.get_sg_connection().base_url
```

Comment:
does this mean that there needs to be a connection in order to get the url? Didn't we introduce a convienence method that allowed you to just pull this from the config file directly to avoid an overhead? Or maybe that doesn't matter in this case?